Google.PowerShell.IntegrationTests/BigQuery/BigQuery.GcbqDataset.Tests.ps1

```diff
@@ -0,0 +1,93 @@
+﻿. $PSScriptRoot\..\BigQuery\GcbqCmdlets.ps1
+$project, $zone, $oldActiveConfig, $configName = Set-GCloudConfig
+
+Describe "New-GcbqDataset" {
+
+	It "should take a name, description, and time to make a dataset" {
+		$data = New-GcbqDataset "test_data_id1" -Name "Testdata" -Description "Some interesting data!" -Timeout 86400000
```

Comment:
Nit: Consider creating a local variable to demystify "86400000". e.g. `$oneDay = ...`.